README.md

```diff
@@ -72,6 +72,7 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/registry/bl
 - [expressive](https://github.com/jinjor/deno-playground/tree/master/expressive) - An express-like web app framework.
 - [flags](https://github.com/denoland/deno_std/tree/master/flags) - Command line arguments parser for Deno based on minimist.
 - [http](https://github.com/denoland/deno_std/tree/master/http) - HTTP module including a file server.
+- [http-libs](https://github.com/denoserverless/http-libs) - HTTP modules and typings for Deno
```

Comment:
nit: Remove `for Deno` and add trailing `.`

Reply:
done